Model

bax45 opened the pull request.

bax45 committed:
3d model spawn

corrected position of the model when it spawns. also it rotates which makes difficult to position furniture so i removed it

bax45 committed:
model optimization

bax45 committed:
Merge branch 'main' into model

parksb commented:
The conflicts seems to have been resolved incorrectly. If you merge this pull request, some of the main changes will be missing😱

bax45 commented:
i made changes only in spawner code to align models as well as in images and their names and deleted floorB and maybe D. Are we using them? If you want to instantiate models that i added you should change in ReferenceImageLibrary names of the images should be same as models'

parksb commented on Assets/Scripts/ARObjectSpawner.cs:
```diff
@@ -77,18 +77,16 @@ private void UpdateSpawned(ARTrackedImage image)
             {
-                spawned.gameObject.transform.rotation = image.transform.rotation;
-            }
-            else
+                spawned.gameObject.transform.rotation = Quaternion.Euler(new Vector3(0, 180f, 0));
+            }if (name == "floor_plan1")
             {
-                spawned.gameObject.transform.rotation = Quaternion.Euler(0, image.transform.rotation.eulerAngles.y, 0);
+                Debug.Log(image.transform.position);
+                spawned.gameObject.transform.rotation = Quaternion.Euler(new Vector3(0, 20f, 0));
             }
 
+
```
I don't understand why this change is needed, the object should rotate according to the image.

And you corrected the angle according to the object, can't you change the default orientation of the prefab itself?

parksb commented on Assets/RefImageLib/KakaoTalk_20240427_192344409.jpg:
Why not use the FloorB image we already have? If you need to add a new one, I think it's better to rename the file to FloorB.jpg.

parksb commented on Assets/RefImageLib/floor_plan1.png:
Isn't this image scoring too low to be used?

parksb commented on Assets/RefImageLib/floor_plan2.png:
Why not use the existing image FloorA.jpg for this as well?

parksb commented:
> i made changes only in spawner code to align models as well as in images and their names and deleted floorB and maybe D. Are we using them? If you want to instantiate models that i added you should change in ReferenceImageLibrary names of the images should be same as models'

I see, I left some comments
